chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1209,13 +1209,22 @@ impl PeerManagerActor {
         );
     }
 
-    fn verify_edges(&mut self, ctx: &mut Context<Self>, peer_id: PeerId, edges: Vec<Edge>) {
+    /// Sends list of edges to check their signatures to `EdgeVerifierActor`.
+    /// Bans peers if an invalid edge is found.
```

Comment:
so these are the edges that we've received from this peer_id, right?

And we'll ban this peer (singular) - if signature on any of the edges is invalid.

Reply:
Yes, I updated the documentation to make it clear.